dev-packages/electron/package.json

```diff
@@ -15,12 +15,19 @@
   },
   "homepage": "https://github.com/theia-ide/theia",
   "bin": {
-    "electron": "cli.js"
+    "electron": "electron-cli.js",
+    "electron-h264-test": "electron-h264-test.js",
+    "electron-replace-ffmpeg": "electron-replace-ffmpeg.js"
   },
   "dependencies": {
     "electron": "^3.1.7",
+    "electron-download": "^4.1.1",
     "electron-store": "^2.0.0",
     "fix-path": "^2.1.0",
-    "native-keymap": "^1.2.5"
+    "native-keymap": "^1.2.5",
+    "unzipper": "^0.9.11"
+  },
+  "scripts": {
+    "postinstall": "node electron-replace-ffmpeg && node electron-h264-test"
```

Comment:
Are you planning distribute and run tests on each install by clients?

Reply:
In this case yes, but should we rather let the clients run the scripts?

I did that so that when people use our `@theia/electron` extension, the pulled Electron binaries are IP clean. But I can change the way it works, wdyt?

Reply:
cc @kittaakos @spoenemann @JanKoehnlein @thegecko any concerns for your electron app? do you want it to be executed by default?

Reply:
Also note that while this works for development, I still need to take a look at the `theia-apps` repository to make `electron-builder` do the same replacement when bundling.

This commit only patches the Electron binaries pulled by the `electron` npm package, somewhere under `node_modules/electron/dist/*`. IIRC `electron-builder` pulls its own Electron distribution, unless configured otherwise.

Reply:
Do you mean this PR still does not prevent from packaging `ffmpeg` in distributed app and each client still has to replace it during bundling himself? Is it really an issue to use `ffmpeg` during development?

Reply:
> Is it really an issue to use `ffmpeg` during development?

I was asking about this too, but I got no response yet :(

On the other hand, I saw an option for `electron-builder` that we use in theia-apps to specify which Electron distribution to use. Since this PR fixes the Electron version in the node modules, I was hoping we could point to that.

Reply:
See this as a preemptive shot to IP clear the source repository, regarding Electron.

Since we had issues about using Electron and whatnot, I wanted to make sure that the way it is used inside this repository is clean (since we also want to move under the EF).

Reply:
Update: I kept working on `theia-apps`, and it is indeed possible to use the same Electron distribution, which means it will bundle using the "clean" ffmpeg library :)

Reply:
This is an awesome change and from our point of view we would like the `ffmpeg` library without proprietary codecs installed automatically by default where possible.

Reply:
> > Is it really an issue to use `ffmpeg` during development?
> 
> I was asking about this too, but I received no response yet :(

We have confirmed with Sharon that it is, for the Foundation. Or rather it's not a problem necessarily for us as such, but rather that a "naive" Theia developer would unknowingly be pulling/using the version of libffmpeg that contains proprietary codecs.